litecli: build fails due to failing test

### Steps To Reproduce
Steps to reproduce the behavior:
1. build **`litecli`**

### Build log

https://gist.github.com/sumnerevans/e4eb46851b21507e4933fa672b0938c6

<details>
<summary>most relevant portion (test output)</summary>


```
============================= test session starts ==============================
platform linux -- Python 3.10.9, pytest-7.2.0, pluggy-1.0.0
rootdir: /build/litecli-1.8.0, configfile: setup.cfg
collecting ... ^Mcollecting 155 items                                                           ^Mcollected 203 items

litecli/packages/completion_engine.py .                                  [  0%]
litecli/packages/special/utils.py .                                      [  0%]
tests/test_clistyle.py ss                                                [  1%]
tests/test_completion_engine.py ........................................ [ 21%]
.....................x..............................................     [ 55%]
tests/test_completion_refresher.py ....                                  [ 57%]
tests/test_dbspecial.py .....                                            [ 59%]
tests/test_main.py ..........                                            [ 64%]
tests/test_parseutils.py .............X.........                         [ 75%]
tests/test_prompt_utils.py .                                             [ 76%]
tests/test_smart_completion_public_schema_only.py ................F..... [ 87%]
                                                                         [ 87%]
tests/test_sqlexecute.py ..................../build/litecli-1.8.0/tests
/build/litecli-1.8.0
......                      [100%]

=================================== FAILURES ===================================
_________________________ test_auto_escaped_col_names __________________________

completer = <litecli.sqlcompleter.SQLCompleter object at 0x7ffff5074fa0>
complete_event = <Mock id='140737304290992'>

    def test_auto_escaped_col_names(completer, complete_event):
        text = "SELECT  from `select`"
        position = len("SELECT ")
        result = list(
            completer.get_completions(
                Document(text=text, cursor_position=position), complete_event
            )
        )
>       assert (
            result
            == [
                Completion(text="*", start_position=0),
                Completion(text="`ABC`", start_position=0),
                Completion(text="`insert`", start_position=0),
                Completion(text="id", start_position=0),
            ]
            + list(map(Completion, completer.functions))
            + [Completion(text="`select`", start_position=0)]
            + list(map(Completion, sorted(completer.keywords)))
        )
E       AssertionError: assert [Completion(t...AVG')])), ...] == [Completion(t...AVG')])), ...]
E         At index 72 diff: Completion(text='select', start_position=0, display=FormattedText([('', 'select')])) != Completion(text='`select`', start_position=0, display=FormattedT>
E         Use -v to get more diff

complete_event = <Mock id='140737304290992'>
completer  = <litecli.sqlcompleter.SQLCompleter object at 0x7ffff5074fa0>
position   = 7
result     = [Completion(text='*', start_position=0, display=FormattedText([('', '*')])), Completion(text='`ABC`', start_position=0...lay=FormattedText([('', 'ABS')])), Completion(>
text       = 'SELECT  from `select`'

tests/test_smart_completion_public_schema_only.py:370: AssertionError
```

</details>

### Additional context
I couldn't find any reports of this error on the upstream repository.

### Notify maintainers
@Scriptkiddi 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.3, NixOS, 23.05 (Stoat), 23.05pre443927.6dccdc45851`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.12.0`
 - channels(root): `"nixos"`
 - channels(sumner): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

--- a/pkgs/development/tools/database/litecli/default.nix
+++ b/pkgs/development/tools/database/litecli/default.nix
@@ -28,6 +28,10 @@ python3Packages.buildPythonApplication rec {
 
   pythonImportsCheck = [ "litecli" ];
 
+  disabledTests = [
+    "test_auto_escaped_col_names"
+  ];
+
   meta = with lib; {
     description = "Command-line interface for SQLite";
     longDescription = ''

